README.md

```
@@ -103,7 +103,7 @@ Alternatively, share your ideas for the plugin over in [discussions](https://git
 | NetBox Version | Plugin Version |
 |----------------|----------------|
 |     4.1        | 0.2.x - 0.3.5  |
-|     4.2 - 4.4       | 0.3.6+          |
+|     4.2 - 4.5       | 0.3.6+          |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)